docs/api/rest_api/rest_api_reference/input/ez-product-products.raml

```diff
@@ -95,6 +95,163 @@
                 description: Error - a product with the same code already exists.
             404:
                 description: Error - no product with the given code exists.
+    /prices:
+        get:
+            displayName: Load the list of product prices
+            description: Loads the list of product prices.
+            headers:
+                Accept:
+                    description: PriceList - if set, the product price list is returned in JSON format.
+                    example:
+                        application/vnd.ibexa.api.PriceList+json
+            responses:
+                200:
+                    description: OK - loads the list of product prices.
+                    body:
+                        application/vnd.ibexa.api.PriceList+json:
+                            type: PriceList
+                            example: !include examples/product/catalog/product/code/prices/GET/PriceList.json.example
+                401:
+                    description: Error - the user has no permission to read prices.
+        post:
```

Comment:
Shouldn't it be per specific product code? The actual URL is `products/{productCode}/prices`.

Reply:
@konradoboza 

- This `/prices` is indented under `/{code}` so this is `/{code}/prices`
- ez-product-products.raml is included under `/product/catalog/products`

Final route n the reference for this line is `POST /product/catalog/products/{code}/prices`

Reply:
Fine, wanted to double-check. RAML isn't super readable to me, frankly speaking. 😅